pkgs/development/libraries/libfive/default.nix

```
@@ -17,7 +17,7 @@
 
 stdenv.mkDerivation {
   pname = "libfive";
-  version = "unstable-2023-06-07";
+  version = "unstable-2024-02-14";
```

```suggestion
  version = "0-unstable-2024-02-14";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

Resolved - didn't know about this.